src/core/public/core_app/status/lib/load_status.ts

```diff
@@ -60,33 +65,60 @@ function formatMetrics({ metrics }: StatusResponse): Metric[] {
       value: metrics.process.memory.heap.used_in_bytes,
       type: 'byte',
     },
+    {
+      name: i18n.translate('core.statusPage.metricsTiles.columns.requestsPerSecHeader', {
+        defaultMessage: 'Requests per second',
+      }),
+      value: (metrics.requests.total * 1000) / metrics.collection_interval_in_millis,
+      type: 'float',
+    },
     {
       name: i18n.translate('core.statusPage.metricsTiles.columns.loadHeader', {
         defaultMessage: 'Load',
       }),
       value: [metrics.os.load['1m'], metrics.os.load['5m'], metrics.os.load['15m']],
       type: 'float',
+      meta: {
+        description: i18n.translate('core.statusPage.metricsTiles.columns.load.metaHeader', {
+          defaultMessage: 'Load interval',
+        }),
+        value: Object.keys(metrics.os.load),
+      },
     },
     {
-      name: i18n.translate('core.statusPage.metricsTiles.columns.resTimeAvgHeader', {
-        defaultMessage: 'Response time avg',
+      name: i18n.translate('core.statusPage.metricsTiles.columns.processDelayHeader', {
+        defaultMessage: 'Delay',
```

Comment:
We didn't have the `Delay` in the status page.

Reply:
Just forgot to add them, likely.